q2_sample_classifier/classify.py

```diff
@@ -323,6 +323,68 @@ def regress_samples_ncv(
     return y_pred, importances
 
 
+def regress_samples_ncv_piepline(
+        ctx, table: biom.Table, metadata: qiime2.NumericMetadataColumn,
+        test_size=defaults['test_size'], step=defaults['step'],
+        cv: int = defaults['cv'], random_state: int = None,
+        n_jobs: int = defaults['n_jobs'],
+        n_estimators: int = defaults['n_estimators'],
+        estimator: str = defaults['estimator_r'], stratify: str = False,
+        parameter_tuning: bool = False,
+        missing_samples: str = defaults['missing_samples']
+        ) -> (pd.Series, pd.DataFrame):
+
+    y_pred, importances, probabilities = nested_cross_validation(
```

Comment:
get action `classify_samples_ncv`, do not call `nested_cross_validation` directly.